package/template-cartesian-graph/package

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(template-cartesian-graph)
+pkgdesc="A Cartesian graph template for the reMarkable tablet"
+url=https://github.com/PeterGrace/template-cartesian-graph
+pkgver=1.0.0-4
+timestamp=2021-01-14T17:46Z
+section=utils
```

Comment:
I think it would be useful to create a section specific to template management.
```suggestion
section=templates
```

Reply:
Which means we should update the doc as part of this PR.

Reply:
Relevant docs are in <https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#section> for reference.